src/express-restify-mongoose.js

```diff
@@ -19,7 +19,8 @@ function getDefaults() {
     runValidators: false,
     allowRegex: true,
     private: [],
-    protected: []
+    protected: [],
+    generateMethods: ['delete', 'deleteone', 'get', 'getone', 'getcount', 'getshallow', 'patch', 'put', 'post', 'modifyobject']
```

Comment:
Codacy has a fix for the issue: [Insert `,`](https://app.codacy.com/gh/florianholzapfel/express-restify-mongoose/pullRequest?prid=8423228)
```suggestion
    generateMethods: ['delete', 'deleteone', 'get', 'getone', 'getcount', 'getshallow', 'patch', 'put', 'post', 'modifyobject'],
```